src/redux/selectors/meta/get_market_by_symbol.js

```diff
@@ -0,0 +1,8 @@
+import { REDUCER_PATHS } from '../../config'
+
+const path = REDUCER_PATHS.META
+
+export default (state, symbol) => {
+  const { markets } = state[path]
+  return markets.find(market => market.wsID === symbol)
+}
```

Comment:
I think it would be better refactor this part with `lodash/find` helper here for app stability, as Bhoomi proposed some time ago

```suggestion
import _find from 'lodash/find'
import { REDUCER_PATHS } from '../../config'

const path = REDUCER_PATHS.META

export default (state, symbol) => {
  const { markets } = state[path]
  return _find(markets, market => market.wsID === symbol)
}
```